dipy/tracking/localtrack.pyx

```diff
@@ -54,7 +54,7 @@ def local_tracker(
         Ending state of the streamlines as determined by the StoppingCriterion.
     """
     cdef:
-        cnp.npy_intp i
+        int i
```

Comment:
why this change? 

it is recommend to use npy_intp variables to iterate over numpy arrays instead of int and size_t.

> "An initial attempt to implement this feature tried to use size_t. It
quickly turned out that this cannot work: Python uses negative indices
in many places (to indicate counting from the end). Even in places where
size_t would be usable, too many reformulations of code where necessary,
e.g. in loops like: `for(index = length-1; index >= 0; index--)` This loop will never terminate if index is changed from int to size_t."

Reply:
ok. I revert it.